src/styles/index.scss

```diff
@@ -1,3 +1,20 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/stars';
+
 body {
   margin: 0;
+  font-family: $primary-font;
+  font-weight: 400;
+}
+
+ul {
```

Comment:
- Don't use tag names for styling (except html and body)
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/24a74c42-9582-4d1a-9eca-ee71cb0c6bc8)